.ansible-lint

```diff
@@ -38,5 +38,9 @@ warn_list:
   - git-latest
   - experimetal  # experimental is included in the implicit list
 
-# offline mode disables installation of requirements.yml
+# Offline mode disables installation of requirements.yml
 offline: false
+
+# Define required Ansible's variables to satisfy syntax check
+extra_vars:
+  - foo: bar
```

Comment:
This is not how vars are defined in ansible, you mush use a dictionary, not a list of dictionaries for variables, somethng like
```
extra_vars:
  foo: bar
  zoo: moo
```